src/instruments/src/EFB/Settings/Pages/RealismPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { usePersistentProperty } from '@instruments/common/persistence';
+import { useSimVar } from '@instruments/common/simVars';
+
+import { Toggle } from '../../UtilComponents/Form/Toggle';
+import { ButtonType, SettingItem, SettingsPage } from '../Settings';
+
+import { SelectGroup, SelectItem } from '../../UtilComponents/Form/Select';
+import { SimpleInput } from '../../UtilComponents/Form/SimpleInput/SimpleInput';
+
+type SimVarButton = {
+    simVarValue: number,
+}
+
+export const RealismPage = () => {
+    const [adirsAlignTime, setAdirsAlignTime] = usePersistentProperty('CONFIG_ALIGN_TIME', 'REAL');
+    const [, setAdirsAlignTimeSimVar] = useSimVar('L:A32NX_CONFIG_ADIRS_IR_ALIGN_TIME', 'Enum', Number.MAX_SAFE_INTEGER);
+    const [dmcSelfTestTime, setDmcSelfTestTime] = usePersistentProperty('CONFIG_SELF_TEST_TIME', '12');
+    const [mcduInput, setMcduInput] = usePersistentProperty('MCDU_KB_INPUT', 'DISABLED');
+    const [mcduTimeout, setMcduTimeout] = usePersistentProperty('CONFIG_MCDU_KB_TIMEOUT', '60');
+    const [boardingRate, setBoardingRate] = usePersistentProperty('CONFIG_BOARDING_RATE', 'REAL');
+    const [realisticTiller, setRealisticTiller] = usePersistentProperty('REALISTIC_TILLER_ENABLED', '0');
+
+    const adirsAlignTimeButtons: (ButtonType & SimVarButton)[] = [
+        { name: 'Instant', setting: 'INSTANT', simVarValue: 1 },
+        { name: 'Fast', setting: 'FAST', simVarValue: 2 },
+        { name: 'Real', setting: 'REAL', simVarValue: 0 },
+    ];
+
+    const dmcSelfTestTimeButtons: ButtonType[] = [
+        { name: 'Instant', setting: '0' },
+        { name: 'Fast', setting: '5' },
+        { name: 'Real', setting: '12' },
+    ];
+
+    const boardingRateButtons: ButtonType[] = [
+        { name: 'Instant', setting: 'INSTANT' },
+        { name: 'Fast', setting: 'FAST' },
+        { name: 'Real', setting: 'REAL' },
+    ];
+
+    const steeringSeparationButtons: (ButtonType & SimVarButton)[] = [
+        { name: 'Disabled', setting: '0', simVarValue: 0 },
+        { name: 'Enabled', setting: '1', simVarValue: 1 },
+    ];
+
+    return (
+        <>
```

Comment:
Remove these React fragments

Reply:
But they were there before and they are need, right? I move the once for the realism settings here and removed them in SimOptions.

Reply:
No, they were needed in the Sim Options page because of the conditional rendering.